docs/quick-ingestion-guides/snowflake/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Overview
+---
+# Snowflake Ingestion Guide: Overview
+
+## What You Will Get Out of This Guide
+
+This guide will help you set up the Snowflake connector through the DataHub UI to begin ingesting metadata into DataHub.
```

Comment:
Let's drop any notion of the "UI". For most people, the UI will be synonymous with DataHub